src/CNTextInput.js

```diff
@@ -1306,7 +1306,7 @@ class CNTextInput extends Component {
 
     render() {
       const {
-        items, foreColor, style, returnKeyType, styleList
+        items, foreColor, style, returnKeyType, styleList, autoCorrect
```

Comment:
`const { items, foreColor, style, returnKeyType, styleList, ...restProps} = this.props`